src/nametables/game_type_menu.js

```diff
@@ -46,6 +46,7 @@ drawTiles(buffer, lookup, `
 #a    T-SPINS                 d#
 #a    SEED                    d#
 #a    STACKING                d#
+#a    MARATHON                d#
```

Comment:
I cant explain why but my headcanon is that marathon should appear right after transition